extension/goext/const.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2017 NTT Innovation Institute, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package goext
```

Comment:
What is the difference between `goext` and `goplugin` packages?

Reply:
goext package is clean in the sense that is contains only interfaces and does not require gohan internals. It allows go plugin to separate from gohan core and in fact it allows go plugins to work. goplugin package just contain the implementation of those plugins. It must be in separate package so that gohan internals will not leak into plugins namespace.